reference/forms/types/collection.rst

```diff
@@ -53,13 +54,9 @@ in a form. For example, suppose you have an ``emails`` field that corresponds
 to an array of email addresses. In the form, you want to expose each email
 address as its own input text box::
 
-    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
-    use Symfony\Component\Form\Extension\Core\Type\EmailType;
-    // ...
-
-    $builder->add('emails', CollectionType::class, array(
+    $builder->add('emails', 'Symfony\Component\Form\Extension\Core\Type\CollectionType', array(
```

Comment:
@weaverryan, the discussion with @xabbuh is on the outdated diff: https://github.com/symfony/symfony-docs/pull/6450#discussion_r59172548
Because 2.8 supports PHP < 5.5, I made a different commit for those changes so it can be reverted in 3.0.
Thanks :)